shared/menubar/index.desktop.js

```diff
@@ -140,7 +140,7 @@ class MenubarRender extends React.Component<Props, State> {
       },
       'Divider',
       ...(this.props.loggedIn ? [{title: 'Open main app', onClick: () => this.props.openApp()}] : []),
-      {title: 'Open folders', onClick: () => this.props.openApp(Tabs.fsTab)},
+      {title: 'Open files', onClick: () => this.props.openApp(Tabs.fsTab)},
```

Comment:
I wonder if this should be `Open Files`. `Files` isn't a word but the name of a tab here, so I don't think it'd contradict to Cécile's preference of not capitalize every word. No strong opinion though, so feel free to leave it!

Reply:
I'm going to leave it for now. We can always change it later.